server/app/routes/api/reviews.js

```diff
@@ -1,20 +1,69 @@
 var express = require('express');
-var router = express.Router();
+var router = express.Router({ mergeParams: true });
 var mongoose = require('mongoose');
 
+require('../../../db/models/review');
+var Review = mongoose.model('Review');
+
+
 // available to eveyone
-router.get('/:id/reviews', function (req, res) {
+router.get('/', function (req, res, next) {
   var id = req.params.id;
-  mongoose.model('User')
-  .findById(id)
-  .then(function(user) {
-    return user.findReviews()
-  })
+  Review.find({author: id})
   .then(function(reviews) {
     res.json(reviews)
   })
-  .then(null, function(err) {
-    console.log(err);
-    res.sendStatus(404);
+  .then(null, next)
+})
+
+router.post('/', function(req, res, next) {
+  Review.create(req.body)
+  .then(function(review){
+    if (review.author === req.params.id) {
+      res.json(review);
+    } else {
```

Comment:
Is this assuming that someone is trying to post a review under another user's account? I'm not sure if we need this if we do something with authentication where a user can't even access users/:id route unless they are an admin or an authorized user of that account. Not sure yet how we will implement that, but we left a comment on the users route file specifying that the router.get('/:id...') should only be available to authorized users.